serveradmin/resources/views.py

```diff
@@ -7,7 +7,7 @@
 from django.views.decorators.csrf import ensure_csrf_cookie
 from django.conf import settings
 
-from adminapi.utils.parse import ParseQueryError, parse_query
+from adminapi.parse import ParseQueryError, parse_query
```

Comment:
I guess you did rename `ParseQueryError` to `QueryError` and moved it to `adminapi.base` in a former commit, didn't you?

Reply:
https://github.com/innogames/serveradmin/pull/5/commits/b68f727d1a749a140632c39609c36895ed70fff7

Seems so, these changes should be in that commit too then. Seems you forgot to refactor these imports here then.

Reply:
The other way around, so it is fine.